assets/css/_mixins.scss

```diff
@@ -29,3 +29,33 @@
     // There is no -moz- version
           touch-callout: default;
 }
+
+// Shared style mixins
+
+// Position and layout for error, loading, and blocking shield
+@mixin blocking-shield {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  padding: 50px;
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
Yes, a nested `div` with table display is the only way, pre-Flexbox, to do vertical centering. Our browser support requirements are pretty lax (last 2 recent browsers of Chrome, Firefox and Safari; IE11 and Edge) so there's no reason not to use Flexbox to simplify markup.